packages/@tinacms/graphql/src/database/index.ts

```diff
@@ -11,65 +11,58 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import _ from 'lodash'
-import * as yup from 'yup'
-import matter from 'gray-matter'
 import path from 'path'
-import { NAMER } from '../ast-builder'
-import { Bridge, FilesystemBridge } from './bridge'
+import { GraphQLError } from 'graphql'
 import { createSchema } from '../schema'
-import { assertShape, lastItem } from '../util'
+import { lastItem } from '../util'
+import { parseFile, stringifyFile } from './util'
+import { sequential } from '../util'
 
+import type { DocumentNode } from 'graphql'
 import type { TinaSchema } from '../schema'
-import type { TinaCloudSchemaBase } from '../types'
-import { DocumentNode, GraphQLError } from 'graphql'
+import type {
+  TinaCloudSchemaBase,
+  TinaFieldEnriched,
+  Collectable,
+} from '../types'
+import type { Store } from './store'
+import type { Bridge } from './bridge'
+import { flatten, isBoolean } from 'lodash'
 
-type CreateDatabase = { rootPath?: string; bridge?: Bridge }
+type CreateDatabase = { bridge: Bridge; store: Store }
 
 export const createDatabase = async (config: CreateDatabase) => {
-  return new Database(config)
+  return new Database({
+    ...config,
+    bridge: config.bridge,
+    store: config.store,
+  })
 }
-
 const SYSTEM_FILES = ['_schema', '_graphql', '_lookup']
 const GENERATED_FOLDER = path.join('.tina', '__generated__')
 
 export class Database {
   public bridge: Bridge
+  public store: Store
   private tinaSchema: TinaSchema | undefined
   private _lookup: { [returnType: string]: LookupMapType } | undefined
   private _graphql: DocumentNode | undefined
   private _tinaSchema: TinaCloudSchemaBase | undefined
   constructor(public config: CreateDatabase) {
-    this.bridge = config.bridge || new FilesystemBridge(config.rootPath || '')
+    this.bridge = config.bridge
+    this.store = config.store
   }
 
   public get = async <T extends object>(filepath: string): Promise<T> => {
     if (SYSTEM_FILES.includes(filepath)) {
-      try {
-        const dataString = await this.bridge.get(
-          path.join(GENERATED_FOLDER, `${filepath}.json`)
-        )
-        return JSON.parse(dataString)
-      } catch (err) {
-        /** File Not Found */
-        if (err instanceof GraphQLError && err.extensions?.status === 404) {
-          throw new GraphQLError(
-            `${err.toString()}.  Please confirm this location is correct and contains a '.tina' folder and a valid '.tina/schema.ts'.`
-          )
-          /** Other Errors */
-        } else {
-          throw err
-        }
-      }
+      throw new Error(`Unexpected get for config file ${filepath}`)
```

Comment:
`get` should only be used for _content_ data, we have system-specific functions like `getTinaSchema` instead.